auth/service_test.go

```diff
@@ -231,9 +248,15 @@ func TestIssue(t *testing.T) {
 				ObjectType:  auth.DomainType,
 				Permission:  auth.MembershipPermission,
 			},
+			checkPolicyRequest2: auth.PolicyReq{
```

Comment:
Rename `checkPolicyRequest` fields to add more context. For example `checkPolicyReq` and `checkPolicyWithStatusReq`.